tests/test_build_featurizer.py

```diff
@@ -250,16 +249,21 @@ def test_initialize_model(model_str, expected_layers, test_size):
             raise AssertionError('Problem loading SqueezeNet weights.')
         check_model_equal(model, model_downloaded_weights)
 
-    assert len(model.layers) == expected_layers
+    # Versions of Keras 2.1.5 and later sometimes use different numbers of layers for these models,
+    # without changing any behavior for predictions.
+    # This checks that the model uses at least one of the expected numbers of layers.
+    assert len(model.layers) in expected_layers
 
-    # Create the test array to be predicted on
```

Comment:
No changes in behavior here, just more descriptive/accurate code, variable names and docstrings